src/course/item.cjsx

```diff
@@ -14,7 +14,7 @@ CourseItem = React.createClass
     routeData = navigation.getDataByView('task')
     return null unless course.isRegistered()
 
-    category = course.catalog_offering_identifier.toLowerCase()
+    category = course.catalog_offering_identifier?.toLowerCase() or 'unknown'
```

Comment:
As I understand it, courses aren't required to have an offering.  They usually will but it's not mandatory.

@jpslav can you confirm?